src/composables/use-usage-session-id.js

```diff
@@ -0,0 +1,22 @@
+import { ssrRef } from '@nuxtjs/composition-api'
+import { v4 as uuidv4 } from 'uuid'
+
+/**
+ * Sets the usageSessionId as a per-request ssrRef. The value is created on the server
+ * in one of the layout components, and is saved to the store to be used when outside
+ * the component context.
+ * TODO: refactor to use Pinia store during the media store refactor.
+ *
+ * @param store
+ * @returns {{ usageSessionId: string }}
+ */
+export function useUsageSessionId(store) {
+  let usageSessionId = ssrRef('')
+  if (!store.state.media.usageSessionId) {
```

Comment:
:grimacing: Would it be better to just create a separate store altogether for it? Throwing unrelated data into the media store doesn't feel right to me.

Reply:
`usageSessionId` is only used in the media store, when sending the `search_query` and `result_clicked` events, so I felt that it was more appropriate to put it to the media store instead of creating a whole store simply for one value. That said, I wonder why `usageSessionId` is not used for other `usage-data` events.

Reply:
> That said, I wonder why usageSessionId is not used for other usage-data events

:thinking: you would think it should be eh?